web/mass_class/rtmp_bridge.py

```
@@ -0,0 +1,178 @@
+# web/mass_class/rtmp_bridge.py
+import asyncio
+import json
+import logging
+import os
+import signal
+import subprocess
```

Ruff (0.8.2)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)